dts/arm/st/c0/stm32c0.dtsi

```diff
@@ -282,6 +282,15 @@
 			vref-channel = <10>;
 		};
 	};
+
+	die_temp: dietemp {
+		compatible = "st,stm32-temp-cal-c0";
+		ts-cal1-addr = <0x1FFF7568>;
+		ts-cal1-temp = <30>;
+		ts-cal-vrefanalog = <3000>;
```

Comment:
Please add the avgslope property here, even if its value is equal to the default one. It will make it clearer and prevent an error if the default is ever modified.